test/e2e-tests.sh

```diff
@@ -53,6 +68,9 @@ function set_feature_gate() {
   fi
   printf "Setting feature gate to %s\n", ${gate}
   jsonpatch=$(printf "{\"data\": {\"enable-api-fields\": \"%s\"}}" $1)
+  if [ "$gate" == "alpha" ]; then
+    jsonpatch=$(printf "{\"data\": {\"enable-api-fields\": \"%s\"}}" $1)
+  fi
```

Comment:
Are you doing anything new here? Doesn't seem like it. Unless I'm missing something obvious.

Reply:
Thats right. It was earlier enabling spire as well, but after discussions with @jerop, decided that we should only enable spire for spire tests.